tests/utils.py

```diff
@@ -12,6 +14,11 @@ def is_fake_datetime(obj):
     return obj.__class__ is FakeDatetime
 
 
+cpython_only_mark = pytest.mark.skipif(
```

Comment:
The `cpython_only` implementation is incompatible with `pytest.mark.parametrize` as far as I can tell, and so I had to use a `pytest` mark instead.

I think that `cpython_only` can be redefined to be this mark, but doing so breaks the `test_utils` test for `cpython_only`, which relies on a certain implementation detail. Personally, I'd rather drop the test for `cpython_only` (rely on coverage metrics to make sure your tests aren't being skipped erroneously) and redefine it as a `pytest` mark, but I wanted to keep the scope of this PR small.